### PR TITLE
GH-2508: Beta version of multitask learning in flair

### PR DESCRIPTION
**Pull request by whoisjones**
Closes gh-2508. Beta version of multitask learning. See tutorial section for more information.

### CODE REVIEW

File: resources/docs/TUTORIAL_11_TRAINING_MULTI_TASK_MODEL.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+supporting (not stable):
+- Combine sequence labeling and text classification tasks
+- Share different embeddings across multiple tasks
+
+What we currently still missing:
+- A proper logging
+- Custom loss averaging functions across multiple tasks
+- Usw Few-Shot Classification or Language Modeling as a task
+
+This features will come but depend on some refactorings which we need
```
Comment (m31415) [inline resources/docs/TUTORIAL_11_TRAINING_MULTI_TASK_MODEL.md:31]:
```suggestion
These features will come but depend on some refactorings which we need
```

**Comment by kiqo** [comment]
hi @whoisjones! I would be interested in the multitask learning, but now see that the PR is closed. Is there a reason why this is now closed?

**Comment by whoisjones** [comment]
Hi @kiqo, we closed this due to a larger refactoring in our model classes. However, I am currently working on multitask learning myself in flair, so I might have some useful branches for you. But it is working only for simple in-domain, encoder-sharing models, and only for sequence labeling and text classification tasks. If this can help you, I can introduce you to the branches.

**Comment by niedakh** [comment]
@whoisjones would you mind writing a small introduction into the branches? I'm using your code in my flair fork and would be very interested on what you're working on.

**Pull request closed without merging**